build/tools/mingw/README.md

```diff
@@ -1,23 +1,38 @@
 MinGW build tools
 -----------------
 
-This folder contains Ms Batch scripts used to create zip packages for officials
-builds with differents versions of the MinGW compiler.
+This folder contains batch files used to create zip packages for official
+builds with different versions of the MinGW compiler.
 
 Building a new release
 ----------------------
 
 1) If needed, update the versions list in the 'buildall.bat' script (the first 'for' loop of this file)
 
 2) Update the 'parameters.bat" file to reflect your system and your installations of the differents MinGW versions. This script is first called without any parameter, to initialize generals values. It will then be called with the compiler version, and the number of bits of the architecture (32 or 64). Here are the lines that will probably need to be changed:
-   * `set JOBS=3` : the value of this variable contains the number of processes that will be launched simultaneously. It is generally recommended to define it as "number of cores of your processor - 1"
+  * `set /A JOBS=%NUMBER_OF_PROCESSORS%-1` : the value of this variable contains the number of processes that will be launched simultaneously. It is automatically calculated as "number of cores of your processor - 1" witch is the recommended value. It can be overriden by writing something like `set JOBS=3` to directly assign a numerical value to this variable
```

Comment:
```suggestion
  * `set /A JOBS=%NUMBER_OF_PROCESSORS%-1` : the value of this variable contains the number of processes that will be launched simultaneously. It is automatically calculated as "number of cores of your processor - 1" which is the recommended value. It can be overridden by writing something like `set JOBS=3` to directly assign a numerical value to this variable.
```